cmd/pruner.go

```diff
@@ -285,12 +285,27 @@ func pruneAppState(home string) error {
 			// custom modules
 			"iscn",     // iscntypes.StoreKey,
 			"nft",      // nftkeeper.StoreKey,
-		        "likenft",  // likenfttypes.StoreKey,
+			"likenft",  // likenfttypes.StoreKey,
```

Comment:
This was my fault, I added it by 4 spaces, now I fix it to tab to align with the style of the full file.